### DISCUSSION TITLE
Can't upload picture for proof of academic status

### DISCUSSION THREAD

**Discussion by AshleyAAabel** [community/community#51871] [GitHub Education]
I am trying to verify my academic status, but there is no way for me to upload a file or screenshot. It appears that my only option is to take a picture with my webcam. The problem with this is that I cannot get all of my required information on one page and the information I need is on my laptop, so taking a picture with my webcam is impractical. Any help is appreciated!

**Comment by samus-aran** [comment]
Hi @AshleyAAabel  👋

We're happy you're here!

It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students submit a live photo of their evidence rather than an upload. We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a photo to capture.

1. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor (a laptop perhaps - or a TV) rather than a phone.
2. If you can, print off your evidence and take a photo of this copy.
3. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.

Let us know here if you need further assistance
